### PR TITLE
Remove temporary files when GKE pods start

### PR DESCRIPTION
**Pull request by michaelkedar**
GKE sends SIGTERM signals to pods when it needs to terminate them, and `tempfile.TemporaryDirectory()` does not clean up its files on the persistent SSD when that happens in its scope.
Manually clear the files when starting the pods.

### CODE REVIEW

**Review by andrewpollock — APPROVED** [review]

File: docker/exporter/exporter.py [hunk 1]
```diff
@@ -118,6 +118,12 @@ def main():
   args = parser.parse_args()
 
   tmp_dir = os.path.join(args.work_dir, 'tmp')
+  # Temp files are on the persistent SSD,
```
Comment (another-rex) [inline docker/exporter/exporter.py:121]:
```suggestion
  # Temp files are on the persistent local SSD,
```

File: docker/importer/importer.py [hunk 2]
```diff
@@ -524,6 +524,12 @@ def main():
   args = parser.parse_args()
 
   tmp_dir = os.path.join(args.work_dir, 'tmp')
+  # Temp files are on the persistent SSD,
```
Comment (another-rex) [inline docker/importer/importer.py:527]:
```suggestion
  # Temp files are on the persistent local SSD,
```

File: docker/worker/worker.py [hunk 3]
```diff
@@ -672,6 +672,12 @@ def main():
   oss_fuzz_dir = os.path.join(args.work_dir, 'oss-fuzz')
 
   tmp_dir = os.path.join(args.work_dir, 'tmp')
+  # Temp files are on the persistent SSD,
```
Comment (another-rex) [inline docker/worker/worker.py:675]:
```suggestion
  # Temp files are on the persistent local SSD,
```

File: docker/exporter/exporter.py [hunk 4]
```diff
@@ -118,6 +118,12 @@ def main():
...
+  # and they do not get removed when GKE sends a SIGTERM to stop the pod.
+  # Manually clear the tmp_dir folder of any leftover files
+  # TODO(michaelkedar): use an ephemeral disk for temp storage.
+  if os.path.exists(tmp_dir):
+    os.rmdir(tmp_dir)
```
Comment (michaelkedar) [inline docker/exporter/exporter.py:126]: wait, this doesn't work if the directory's not empty...

**Review by another-rex — APPROVED** [review]
Just clarify that it's the local SSD

**Review by oliverchang — APPROVED** [review]

**Pull request merged**